core/src/main/java/org/apache/calcite/sql2rel/SqlToRelConverter.java

```diff
@@ -1711,19 +1717,18 @@ public RelNode convertToSingleValueSubq(
   /**
    * Converts "x IN (1, 2, ...)" to "x=1 OR x=2 OR ...".
    *
-   * @param leftKeys   LHS
+   * @param leftSqlKeys   LHS
```

Comment:
does this parameter need to be renamed?

Reply:
not necessarily - I've undone the rename